### PR TITLE
Added panel "The Future of Ember"

### PR DESCRIPTION
**Pull request by emmasofiah2o**
- Added file
- Added image

TO-DO:
- Add description

### CODE REVIEW

**Comment by emmasofiah2o** [comment]
@marcoow the pr is ready. It would be awesome if you could provide a description or a few bullet points as we discussed on slack.

File: src/appearances/2022-09-22-the-future-of-ember.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: "Panel: The Future of Ember"
+image: "/assets/images/talks/2022-09-22-emberfest-2022/the-future-of-ember.jpg"
+url: https://www.youtube.com/watch?v=03Ti0szjNPE
+media: video
+channel: emberfest-2022
+---
```
Comment (marcoow) [inline src/appearances/2022-09-22-the-future-of-ember.md:7]:
```suggestion

---
EmberFest 2022 closed with a panel about the future of Ember, moderated by Marco Otte-Witte with panelists Melanie Sumner, Chris Manson, Ed Faulkner, and Chris Krycho. The panel discussed challenges and opportunities, how we market Ember and where it fits in in the wider ecosystem of frontend frameworks.
```

**Review by marcoow — CHANGES_REQUESTED** [review]
added description

File: src/appearances/2022-09-22-the-future-of-ember.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+title: "Panel: The Future of Ember"
+image: "/assets/images/talks/2022-09-22-emberfest-2022/the-future-of-ember.jpg"
+url: https://www.youtube.com/watch?v=03Ti0szjNPE
+media: video
+channel: emberfest-2022
+
+---
+EmberFest 2022 closed with a panel about the future of Ember, moderated by Marco Otte-Witte with panelists Melanie Sumner, Chris Manson, Ed Faulkner, and Chris Krycho. The panel discussed challenges and opportunities, how we market Ember and where it fits in in the wider ecosystem of frontend frameworks.
```
Comment (emmasofiah2o) [inline src/appearances/2022-09-22-the-future-of-ember.md]:
```suggestion
EmberFest 2022 closed with a panel about the future of Ember, moderated by Marco Otte-Witte with panelists Melanie Sumner, Chris Manson, Ed Faulkner, and Chris Krycho. The panel discussed challenges and opportunities, how we market Ember and where it fits in the wider ecosystem of frontend frameworks.
```

Comment (emmasofiah2o) [reply src/appearances/2022-09-22-the-future-of-ember.md]: @marcoow I think we can leave one "in" out

Comment (marcoow) [reply src/appearances/2022-09-22-the-future-of-ember.md]: ok 👍

**Review by marcoow — APPROVED** [review]

**Pull request merged**